src/app/components/wrapper-components/didacticial-game-wrapper/didacticial-game-wrapper.component.html

```diff
@@ -16,34 +16,34 @@
                         *ngIf="tutorialOver === false && ((stepFinished && stepFinished[stepIndex] === true) || moveAttemptMade)"
                         (click)="retry()"
                         i18n>
-                    Essayez à nouveau
+                    Try again
                 </button>
                 <button id="nextButton"
                         class="button is-primary mr-3 mb-1"
                         *ngIf="tutorialOver === false"
                         (click)="next()"
                         >
-                    <ng-container *ngIf="steps && steps[stepIndex].isInformation() else notInfo" i18n>Vu</ng-container>
+                    <ng-container *ngIf="steps && steps[stepIndex].isInformation() else notInfo" i18n>Ok</ng-container>
                     <ng-template #notInfo>
```

Comment:
why not "seen" ?

Reply:
C'est une question d'idiome: là où « vu » en français a du sens, « seen » me semble très artificiel en anglais